Package.swift

```diff
@@ -1,5 +1,12 @@
 import PackageDescription
 
 let package = Package(
-    name: "CryptoSwift"
+    name: "CryptoSwift",
+    targets: [
+                Target(
+                    name: "CryptoSwiftTests",
+                    dependencies: [.Target(name: "CryptoSwift")]),
```

Comment:
This is not needed, SwiftPM automatically creates a Tests target when there's a folder called `Tests` with `CryptoSwiftTests` inside of it.